pom.xml

```diff
@@ -35,54 +41,7 @@
 		<warContext>JDBCApp</warContext>
 		<package.file>${project.build.directory}/${project.artifactId}.zip</package.file>
 		<packaging.type>usr</packaging.type>
-</properties>
-
-
-	<profiles>
-		<profile>
-			<id>runnable</id>
-			<properties>
-				<package.file>${project.build.directory}/${project.artifactId}.jar</package.file>
-				<packaging.type>runnable</packaging.type>
-			</properties>
-		</profile>
-		<profile>
-			<id>liberty-test</id>
-
-			<!--Do not start the server unless testing -->
-
-			<activation>
-				<property>
-					<name>!skipTests</name>
-				</property>
-			</activation>
-
-			<build>
-				<plugins>
-					<plugin>
-						<groupId>net.wasdev.wlp.maven.plugins</groupId>
-						<artifactId>liberty-maven-plugin</artifactId>
-						<executions>
-							<execution>
-								<id>start-server</id>
-								<phase>pre-integration-test</phase>
-								<goals>
-									<goal>start-server</goal>
-								</goals>
-							</execution>
-							<execution>
-								<id>stop-server</id>
-								<phase>post-integration-test</phase>
-								<goals>
-									<goal>stop-server</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+	</properties>
```

Comment:
Dylan, I wanted to leave in the runnable profile because that makes it very easy for them to switch from producing a zip to producing a runnable.  To test that it works OK with your changes, do a `mvn install -Prunnable`.  Then take the zip produced and from a command line, run `java -jar ${project.artifactId}.jar`.  It should extract the jar and start the server and application.  You should be able to run the app from the browser.